html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
```

Comment:
Inlines script are all well and good for first drafts, but on `master`, they're *no bueno*.  Please put this JS code into a file within the `js` directory.

Reply:
Sure, good call. I've moved this code into the following file: `/js/host/ui/ui-ux-display.js`. Lmk if you'd like to change the name of the file / anything else regarding this comment.